README.md

```diff
@@ -69,15 +69,15 @@ aurelia.use.plugin("@ssv/au-core", config);
 ### Setup Machine for Development
 Install/setup the following:
 
-- NodeJS v7+
+- NodeJS v8+
 - Visual Studio Code or similar code editor
-- TypeScript 2.2+
+- TypeScript 2.5+
 - Git + SourceTree, SmartGit or similar (optional)
 - Ensure to install **global NPM modules** using the following:
 
 
 ```bash
-npm install -g git gulp yarn karma-cli
+npm install -g git gulp yarn jest-cli
```

Comment:
is it necessary `jest-cli` to be installed globally?

Reply:
not really it's not required. I will remove it

Reply:
updated, removed jest-cli from readme